server/src/pages/settings/model.js

```diff
@@ -0,0 +1,26 @@
+const db = require("../../db");
+
+exports.createModel = function(req) {
+  let title = "Settings";
+  let model = {
+    noAnalytics: true,
```

Comment:
Oh, analytics!  I think we should have analytics, and we should add events for connect and disconnect.  I think the default pageview will handle the basic count of how many people visit the settings page.